### PR TITLE
Fixes/20190311

### PR DESCRIPTION
**Pull request by gre**
details are in the commits

### CODE REVIEW

**Comment by gre** [comment]
Everything should be fixed now.

File: src/components/AccountGraphCard.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ class AccountGraphCard extends PureComponent<Props, State> {
           isInteractive={isAvailable}
           isLoading={!isAvailable}
           height={100}
-          width={Dimensions.get("window").width - 40}
+          width={Dimensions.get("window").width - 32}
```
Comment (juan-cortes) [inline src/components/AccountGraphCard.js]: Shouldn't this use `getWindowDimenions`? 
There's some platform specific stuff for android 🤷‍♂️

File: src/components/GraphCard.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ class GraphCard extends PureComponent<Props, State> {
           isInteractive={isAvailable}
           isLoading={!isAvailable}
           height={100}
-          width={getWindowDimensions().width - 40}
+          width={getWindowDimensions().width - 32}
```
Comment (juan-cortes) [inline src/components/GraphCard.js]: same

Comment (gre) [reply src/components/GraphCard.js]: Here it's used right?

**Comment by gre** [comment]
@Arnaud97234 ready for review. the error cases are potential not fixed but it should not be a regression because i'm pretty sure it's as on develop.

**Pull request merged**